code/ac/llama/LogitComparer.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Alpaca Core
+// SPDX-License-Identifier: MIT
+//
+#pragma once
+#include "Token.hpp"
+#include <unordered_map>
+
+namespace ac::llama {
+
+class LogitComparer {
```

Comment:
Maybe we should rename this class since it's not only legit comparison only? Does `TokenDataComparer` sounds better?